python/lsst/ip/diffim/getTemplate.py

```diff
@@ -637,18 +644,23 @@ def run(self, coaddExposures, bbox, wcs):
         tractsCatalog = afwTable.ExposureCatalog(tractsSchema)
 
         finalWcs = wcs
-        bbox.grow(self.config.templateBorderSize)
         finalBBox = bbox
+        tempBBox = geom.Box2I(bbox.getBegin(), bbox.getDimensions())
```

Comment:
Curious why you changed the `template` to have the bbox of the original calexp bbox rather the grown one.